velox/connectors/hive/TableHandle.h

```diff
@@ -115,6 +121,7 @@ class HiveColumnHandle : public ColumnHandle {
   const TypePtr dataType_;
   const TypePtr hiveType_;
   const std::vector<common::Subfield> requiredSubfields_;
+  const std::unordered_map<std::string, std::string> columnParameters_;
```

Comment:
Please add a comment here indicating that these are used for metadata like column date value format. 

If you have a link to how these are populated that will be great as well.

This change would be accompanied by presto-native code to populate these fields in IcebergPrestoToVeloxConnector::toVeloxColumnHandle method. So is there a change in the protocol classes as well to pass these from the Iceberg connector ? Do you the e2e Prestissimo PR ? Would be great to take a look at that to understand the scope of this change.

Reply:
Yes. We will need to change `IcebergPrestoToVeloxConnector::toVeloxColumnHandle`. This change will need to be merged first for me to work on the Prestissimo PR.

Reply:
Would be great to see the Prestissimo code at the same time to know this works e2e.

You can change the Velox submodule in the Presto project to pick up the changes from your branch https://git-scm.com/book/en/v2/Git-Tools-Submodules section "Pulling in Upstream Changes from the Submodule Remote"
and try out this code with presto-native-execution test.

See https://github.com/prestodb/presto/pull/24138 for how a PR looks with Velox submodule changes for a branch with local commits.